EXTENSION_VERSION

```diff
@@ -0,0 +1 @@
+0.5.0
```

Comment:
Nit: new line missing. I guess we need to make sure it parses properly with new line. Maybe better use regex to handle the version since we follow semantic versioning.

Reply:
done